style.css

```diff
@@ -0,0 +1,516 @@
+
+/* Box sizing rules */
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+/* removing default margin */
+
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+figure,
+blockquote,
+dl,
+dd {
+    margin: 0;
+}
+
+/* set core root defaults */
+
+html:focus-wthin {
+    scroll-behavior: smooth;
+}
+
+/* set core body defaults */
+
+body {
+    min-height: 100vh;
+    text-rendering: optimizeSpeed;
+    line-height: 1.5;
+    font-family: "Roboto";
+}
+
+/* a elements that don't have a class get default styles */
+
+a:not([class]) {
+    text-decoration-skip-ink: auto;
+}
+
+/* make images easier to work with */
+
+img,
+picture {
+    max-width: 100%;
+    display: block;
+}
+
+/* inherit fonts for inputs and buttons */
+
+input,
+button,
+textarea,
+select {
+    font: inherit;
+}
+
+/* remove all animations, transitions and smooth scroll for pe @media (prefers-reduced-motions: reduce) */
+
+@media (prefers-reduced-motion: reduce) {
+    
+    html:focus-within {
+        scroll-behavior: auto;
+    }
+
+    *,
+    *::before,
+    *::after {
+        animation-duration: 0.01ms !important;
+        animation-iteration-count: 1 !important;
+        transition-duration: 0.01ms !important;
+        scroll-behavior: auto !important;
+    }
```

Comment:
- Please Avoid overusing  !important.

Reply:
- [ ] [OPTIONAL] - I think, you can describe here in more detail, why it should be avoided overusing `!important`